CONTRIBUTING.md

```diff
@@ -97,7 +97,26 @@ Therefore, please don't use type annotations for class attributes, instead write
 ### Formatting
 
 We use `black` (with line-length 119) and `isort` (black profile) as automated formatters. On pull requests, this is automatically checked and you can only merge if your code passes the check. You can include both in a 
-[pre-commit](https://pre-commit.com/) locally by executing `mamba install pre-commit` and `pre-commit install`, which creates a hook in your local git repository that is called when a commit is executed. 
+[pre-commit](https://pre-commit.com/) locally by executing 
+
+```shell
+mamba install pre-commit
+pre-commit install
+```
+
+which creates a hook in your local git repository that is called when a commit is executed. 
 This will work assuming you have `black` and `isort` installed in your environment. `pre-commit` takes information from the `.pre-commit-config.yaml` to know which actions to perform. The formatting rules for both tools are defined in `pyproject.toml`.
 
 **Note:** We may decide on slight variations in the code formatting. This will be mentioned here in the future.
+
+## Documentation
+
+You should always test if the api documentaion correctly renders the types, parameters, return values and attributes. 
+This can be checked by building the documentation locally. Just run the following commands:
+
+```shell
+cd docs
+make html
+```
+
+Please take note, if this generates or changes files in `docs/api`. If so, commit and push tehm as well, since this cannot be done automatically by readthedocs. The `_build` directory is listed in the `.gitignore` and should not be commited.
```

Comment:
```suggestion
Please take note, this can generates or changes files in `docs/api`. If so, commit and push them as well, since this is currently not included in your workflow and cannot be done automatically by readthedocs. The `_build` directory is listed in the `.gitignore` and should not be commited.
```